### ISSUE TITLE
Diagram renderer should provide scrollbars if diagram is larger than the viewport of editor pane

### ISSUE THREAD

**Issue by Hunter21007** [doctorrustynelson/vscode-nomnoml#2]
Currently if the diagram size is larger than the editor view, it will be cut off and it is not possible to see that is behind viewport's bounds

i have wrote this hacky change to provide scroll bars:

file: extension.js, line: 33

``` javascript
return "<html><body>" + svg +
              "<script>" +
                    "var s = document.getElementsByTagName('svg')[0];" +
                    "var r = s.firstChild;" +
                    "s.setAttribute('width', r.width.animVal.value + 'px');" +
                    "s.setAttribute('height', r.height.animVal.value + 'px');" +
                "</script>" +
            "<body></html>";
```

although jquery would be better solution.

(and sorry for my English)

**Comment by Hunter21007** [comment]
As expected this will no work properly, since afted refresh scrollbars are gone.
so i adjusted my solution:

``` javascript
            return "<html><body style=\"width:100%;height:100%;margin:0px;\">" + 
                "<div style=\"width:100%;height:100%;overflow:scroll;\">" + svg + 
                "<script>" +
                    "var s = document.getElementsByTagName('svg')[0];" +
                    "var r = s.firstChild;" +
                    "s.setAttribute('width', r.width.animVal.value + 'px');" +
                    "s.setAttribute('height', r.height.animVal.value + 'px');" +
                "</script>" +
            "</div></body></html>";
```

**Comment by Hunter21007** [comment]
Works only if the diagram is wrapped by &lt;frame&gt; :)
and it must be first element in code, sorry

**Comment by doctorrustynelson** [comment]
Thank you very much for the example code!  Using it I was able to construct a method by which scrolling will work.  It's now part of the code base and available with version 0.2.0 and higher.
